src/controller/python/chip/clusters/ClusterObjects.py

```diff
@@ -192,12 +258,12 @@ def _cluster_object(cls) -> ClusterObject:
         return make_dataclass('InternalClass',
                               [
                                   ('Value', List[cls.attribute_type.Type]
-                                   if cls.attribute_type.IsArray else cls.attribute_type.Type, field(default=None)),
+                                   if isinstance(cls.attribute_type.Type, List) else cls.attribute_type.Type, field(default=None)),
```

Comment:
Seems we should just remove this line and use cls.attribute_type.Type instead?

BTW, this is not rejected by the unit test since `List[x]` is not a instance of `List`